### DISCUSSION TITLE
Get-PnPUserProfileProperty -account <UserPrincipalName> | Select PersonalUrl  No longer Works with PowerShell 7.x

### DISCUSSION THREAD

**Discussion by circbit** [pnp/powershell#3399] [Q&A]
With Powershell 5.x, Get-PnPUserProfileProperty -account <UserPrincipalName> | Select PersonalUrl would return the User's OneDrive URL. Now with PnP.Powershell 2.2.0 on PowerShell 7.3, "Select" no longer returns anything.

I'm attempting to run the script below, but I think it chokes on this line:
$url=Get-PnPUserProfileProperty -Account $emailAddress | Select PersonalUrl

https://learn.microsoft.com/en-us/purview/ediscovery-use-a-script-to-add-users-to-a-hold#step-1-install-the-sharepoint-online-management-shell

Thanks in advance for any help

**Comment by Nick-MAC** [comment]
This command is now outputting the values toa  hashtable for some reason... Is this a feature update or just a change in the way this command works?
